client/theme.js

```diff
@@ -88,6 +88,13 @@ export default {
     Icon: {
       default: grays.middleGray,
       hover: grays.darker
+    },
+    Panel: {
```

Comment:
These are same colours as the buttons, so we'll have a problem if someone later puts a Button on top of a Panel as they'll be the same.

Shall we take the colours [from the console header component](https://github.com/processing/p5.js-web-editor/blob/develop/client/styles/components/_console.scss)? Cassie made accessibility colour changes a a while back to ensure the contrast worked across all the themes.

So:

`Panel.default.foreground` should be the value of `console-header-color`
`Panel.default.background` should be the value of  `console-header-background-color`
`Panel.default.border` should be the value of  `ide-border-color`

The [colours are here](https://github.com/processing/p5.js-web-editor/blob/67f98246423a9bd87fb671d7f2f1e9929c27c5e8/client/styles/abstracts/_variables.scss).

I know it's a bit of a pain to do this, but it'll really help with the move to styled-components and away from SCSS and helping to make the design more consistent.

Reply:
True, but these are the actual colors of the mobile UI. I think I'll create a `Panel: { mobile: {} }` attribute and keep these colors there, so the default `Panel` colors can be changed.
Do you think that makes sense?

Reply:
If possible, it would be great to have the same set of colours work for mobile and the legacy UI. My feeling is branching the theme into _Panel.mobile_ and _Panel.notMobile_ creates more code for us to maintain.

Your right that maybe this isn't a reusable panel colour, so if it's not possible to find common colours, then my preference is that we name the variable something other than Panel.

@catarak, what do you think?

Reply:
I investigated this a bit, and It's an important decision that goes beyond the scope of the PR, so I'll create an issue and we can discuss it there!

Reply:
Ok, good idea.

So shall we call this MobilePanel or something for now and we can get this merged in?

Reply:
I agree with @andrewn, that it would be helpful if possible to not have mobile specific colors. 

@ghalestrilo and I talked this over and thought it would make sense to open a new issue/PR with respect to the panel and button color contrasts!

Reply:
Nice! I'll update this :)